source/Cosmos.VS.DebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -131,7 +131,14 @@ int IDebugPendingBreakpoint2.Bind()
 
                             // We have the method. Now find out what Sequence Point it belongs to.
                             var xSPs = xDebugInfo.GetSequencePoints(asm.Pathname, xMethod.MethodToken);
-                            var xSP = xSPs.Single(q => q.LineColStart <= xPos && q.LineColEnd >= xPos);
+                            var xSP = xSPs.Single(
```

Comment:
Dd the position stuff suddenly change in the debug info database?

Reply:
Nothing changed in the database, I simply removed `SequencePoint` (https://github.com/CosmosOS/IL2CPU/pull/66/commits/44c5a1dd396497239e6dcaf94fbc3c95b82bfc4c#diff-ad3cca2313932e14cee35379036aa0adL325) as there's a new `SequencePoint` class in the new library, but it doesn't have `LineColStart` nor `LineColEnd`. If you want, I can add extension methods to replace this property or create a wrapper for the new `SequencePoint` class which adds these properties.